test/plugins/stats.test.js

```diff
@@ -19,7 +19,7 @@ describe('stats', function () {
        .defaultReplyHeaders({'Content-Type': 'application/json'})
        .post('/installations/1/access_tokens').reply(200, {token: 'test'})
        .get('/app/installations?per_page=100').reply(200, [{id: 1, account: {login: 'testing'}}])
-       .get('/installation/repositories').reply(200, {repositories: [
+       .get('/installation/repositories?per_page=100').reply(200, {repositories: [
```

Comment:
Was there a reason for this change? I have no problem with it, I'm just curious

Reply:
that is actually a very good question. I’ve run into something similar with Greenkeeper’s test suite: https://github.com/greenkeeperio/greenkeeper/pull/582/files#diff-3fc4b15a6c1901c0b988f4a642877425L16

Maybe it is a regression, I’ll investigate it

Reply:
so it seems to me that this was actually a bug that got fixed now!

Turns out that the `per_page` & `page` params are missing in [routes.json](https://github.com/octokit/rest.js/blob/master/lib/routes.json#L1682-L1687), which means, with the previous implementations, they have been ignored! 

The new implementation automatically passes all remaining options as query parameter (for `GET` requests) so it works anyway.

So tl;dr: the new release actually fixed a bug, getting the stats will be more performant now, as it gets 100 results instead of 30